lint-workflow/lint.py

```diff
@@ -1,6 +1,47 @@
 import argparse
 import os
 import yaml
+import json
+import urllib3 as urllib
+
+
+def get_action_update(action_id):
+    """
+    Takes and action id (bitwarden/gh-actions/version-bump@03ad9a873c39cdc95dd8d77dbbda67f84db43945)
+    and checks the action repo for the newest version.
+    If there is a new version, return the url to the updated version.
+    """
+
+    path, *hash = action_id.split("@")
+    http = urllib.PoolManager()
+    headers = {}
+
+    if os.getenv("GITHUB_PAT", None):
+        headers["Authorization"] = f"Token {os.environ['GITHUB_PAT']}"
```

Comment:
If this env var is not set, will the linter still run without this functionality?

Reply:
Yes, if the environment var isn't found, it defaults with `None`. This allows the linter to still run, but is subject to Github's anonymous rate limits (60/hr).

Reply:
Will this have any affect in running this linter in our linting workflows?

Reply:
I moved to naming it `GITHUB_TOKEN` which should ensure that it uses the default runner PAT.

Reply:
Perfect, thanks!